mitosheet/src/mito/components/toolbar/HomeTabContents.tsx

```diff
@@ -147,7 +147,6 @@ export const HomeTabContents = (
 
         <div className='mito-toolbar-number-format'>
             <Select
-                style={{ height: '22px', fontSize: '11px', width: '150px' }}
```

Comment:
Why does this go away?

Reply:
Oh I added css into `toolbar.css` to handle this. Wanted it to be used for all toolbar selects.